### PR TITLE
Screengif

### PR DESCRIPTION
**Pull request by vltansky**
Generate terminal gif for readme ([example](https://github.com/vltansky/create-html5-boilerplate/tree/screengif)) with [terminalizer](https://terminalizer.com/).

questions:
1. Should we save config & yaml to be able to edit/regenerate gif or just to store gif without configs?
2. Need to find the way to change terminal to bash

### CODE REVIEW

File: imgs/config.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Specify a command to be executed
```
Comment (roblarsen) [inline imgs/config.yml:1]: please just call the folder `img` to map to what we do in the main project

File: imgs/record.md [hunk 2]
```diff
@@ -0,0 +1 @@
+https://terminalizer.com/docs
```
Comment (roblarsen) [inline imgs/record.md:1]: What is this file?

**Comment by roblarsen** [comment]
This is cool. I don't mind powershell if you can't get it to be bash.

File: imgs/record.md (hunk 2 above)
Comment (vltansky) [reply imgs/record.md]: deleted

**Pull request merged**